oaknational/Oak-Web-Application
# Feat/lesq 710/sort specialist units lessons [LESQ-710]

## Description


Music year: 1953

- lift specialist queries to use mv version 1_0_3
- update lesson listing schema with `order_in_unit`
- update unit listing schema with `order_in_programme`
- sort units and lessons using these values

## How to test

1. Go to https://deploy-preview-2335--oak-web-application.netlify.thenational.academy/teachers/specialist/programmes/communication-and-language-applying-learning/units
2. units shold be in the correct order
3. click on unit
4. lessons should be in the correct order

## Screenshots

How it used to look (delete if n/a):
<img width="500" alt="Screenshot 2024-03-27 at 09 40 01" src="https://github.com/oaknational/Oak-Web-Application/assets/45038878/993a95b3-2a69-4d0c-b5f6-64b3106ccea6">
<img width="500" alt="Screenshot 2024-03-27 at 09 40 40" src="https://github.com/oaknational/Oak-Web-Application/assets/45038878/6ce6deb5-6aa9-429f-b017-294c8edfdcf5">

How it should now look:
<img width="500" alt="Screenshot 2024-03-27 at 09 39 33" src="https://github.com/oaknational/Oak-Web-Application/assets/45038878/7e57bcd2-5ee6-481d-9581-ef4336dca946">
<img width="500" alt="Screenshot 2024-03-27 at 09 40 23" src="https://github.com/oaknational/Oak-Web-Application/assets/45038878/7f49c225-5508-4f3c-9da5-d3bb8842cd3c">
